docs/spec/channel.md

```diff
@@ -196,6 +196,14 @@ Note: The array of subscribers MUST NOT be set directly on the generic Channel
 custom object, but rather appended to the backing channel by the subscription
 itself.
 
+#### Channelable and Subscription Delivery Spec
+
+Both Channelable and Subscription has a `Delivery` field of type `duck.DeliverySpec` (per `v1`).
+This field allows the user to define the dead letter sink and retries.
+The `ChannelableSpec.Delivery` field is global across all the Subscriptions registered to that particular
+Channelable, while the `SubscriptionSpec.Delivery`, if configured, fully overrides `ChannelableSpec.Delivery` for
```

Comment:
Do we typically leak implementation details about Go type in spec docs? I see the more generic `spec.xyz` format used in a few places.

Reply:
`+1`

Reply:
Why are we leaking implementation details here? Can you propose an alternative formulation?

Reply:
Do you propose to use `channel.spec.delivery` more than `ChannelableSpec.Delivery` (same for sub)?

Reply:
Yes, as mentioned `ChannelableSpec.Delivery` is Go code. I don't see Go code in the rest of the spec, but rather the user facing API.